app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt

```diff
@@ -68,7 +69,7 @@ class CommonWebViewFragment: Fragment() {
                     request.url.toString().replace(Regex("^.+\\?u=|\\&l.+$"), ""),
                     StandardCharsets.UTF_8.toString()
                 )
-                val uri = Uri.parse(decodedUrl).buildUpon().scheme("http").build()
+                val uri = Uri.parse(decodedUrl).buildUpon().scheme("https").build()
```

Comment:
Sadly, even in 2020, we cannot assume every website / webapp will support https. If the protocol is entered as http:// specifically, we probably need to keep that, otherwise assume https.

Reply:
stopped forcing a scheme for the uri, instead it will use the scheme decoded from the url